provider-tutorials/provider-tutorial.md

```diff
@@ -25,7 +25,7 @@ For this release we have prepared a dedicated and controlled sub-network using E
 {% endhint %}
 
 {% hint style="info" %}
-If you would like to earn real GNTs now, head over to our [Clay Golem Beta implementation](https://golem.network/download/clay-beta/).
+If you would like to earn real GLMs now, head over to our [Clay Golem Beta implementation](https://golem.network/download/clay-beta/).
```

Comment:
in clay you are actually still earning GNT's

Reply:
Do you see the need to update this to more recent version or I should just revert to GNT back?

Reply:
i would either:
- revert back
- add ( pre-migration ) before GNT